msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -47,11 +47,17 @@
 
 /**
  * <p>
- * This is the entry point for developer to create the public native applications and make API call to acquire tokens. MSAL {@link PublicClientApplication} provides three constructors allowing the client id to be set either via AndroidManifest.xml metadata or using constructor parameters.
+ * This is the entry point for developers to create the public native application and make API calls to acquire tokens.
+ * <p><b>Client ID:</b>  The clientID of your application is a unique identifier which can be obtained from the app portal.</p>
+ * <p><b>Authority:</b> A URL indicating a directory that MSAL can use to obtain tokens. In Azure AD
+ * it is of the form https://<instance/<tenant>, where <instance> is the directory host (e.g. https://login.microsoftonline.com)
+ * and <tenant> is an identifier within the directory itself (e.g. a domain associated to the
+ * tenant, such as contoso.onmicrosoft.com, or the GUID representing the  TenantID property of the directory)</p>
+ * MSAL {@link PublicClientApplication} provides three constructors allowing the client id to be set either via AndroidManifest.xml metadata or using constructor parameters.
  * Similarly, if developer chooses not to use the default authority (https://login.microsoftonline.com), an alternate can also be configured using the manifest or constructor parameters.
  * </p>
  * <p>
- * Redirect is auto-generated in the library in the format of msal<client-id>://auth, it cannot be overridden.
+ * Redirect is auto-generated in the library in the format of msal<client-id>://auth, and it cannot be overridden.
```

Comment:
i added "and"

---
In reply to: [114632918](https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/160#discussion_r114632918) [](ancestors = 114632918)